README.md

```diff
@@ -13,6 +13,18 @@ Output:
 Logs "Hello!" to the console
 ```
 
+<!-- console.log("question one")
+
+const sayHello = (str) => {
+return "hello!"
+}
+
+const callBack = (fn) => {
+    return sayHello()
+}
```

Comment:
You want to be calling the fn that you passed in. The fn passed in IS the callback. Calling sayHello() is not using the callback